### PR TITLE
add dedicated to identify intensive requests that should be processed…

### PR DESCRIPTION
**Pull request by tonyfujs**
Some queries are more compute intensive than others, and should be forked into a dedicated parallel process in order to avoid blocking the main API R process.

These queries were previously identified in an ad hoc fashion in the `endpoints.R` file. I created a dedicated, unit tested function that takes care of idenitfying slow requests.

### CODE REVIEW

**Review by randrescastaneda — APPROVED** [review]
I think everything looks fine. However, besides the fact that it is more organized and standardized across all endpoints, Is there something different that this function does that was not being done before? Also, The intensity threshold is a parameter, but I don't see how we can modify it or when it should be modified.

**Comment by tonyfujs** [comment]
> I think everything looks fine. However, besides the fact that it is more organized and standardized across all endpoints, Is there something different that this function does that was not being done before? 

The overall logic is the same but it does a few things a bit better: 
- "WLD" was not controlled for before.
- The previous way of checking things would discard elements if country vectors were of length > 1. For instance `country = C("COL", "ALL")` would not have been identified as an intensive query. ONly `country = "COL"` would have been kept.

> Also, The intensity threshold is a parameter, but I don't see how we can modify it or when it should be modified.
Yes. That is on purpose. It is not supposed to be modified anywhere else than here. I could have hard coded it in the function as well... This might not be the best implementation though, so we can revisit later if needed.

File: inst/plumber/v1/endpoints.R [hunk 1]
```diff
@@ -223,6 +223,7 @@ function(req, res) {
 #* Default is FALSE
 
 function(req, res) {
+  browser()
```
Comment (shahronak47) [inline inst/plumber/v1/endpoints.R:226]: Hi @tonyfujs , I think we missed in a `browser()` statement here.

Comment (tonyfujs) [reply inst/plumber/v1/endpoints.R:226]: Good catch! Thanks @shahronak47

**Pull request merged**